go/stats/metrics.go

```diff
@@ -0,0 +1,69 @@
+package main
+
+import (
+	"context"
+	"database/sql"
+
+	_ "github.com/go-sql-driver/mysql"
+	"github.com/inverse-inc/packetfence/go/interval"
+	"github.com/inverse-inc/packetfence/go/log"
+	"github.com/inverse-inc/packetfence/go/pfconfigdriver"
+)
+
+var MySQLdatabase *sql.DB
+
+func ProcessMetricConfig(ctx context.Context, conf pfconfigdriver.PfStats) error {
+
+	// Read DB config
+	configDatabase := readDBConfig()
+	connectDB(configDatabase, MySQLdatabase)
+	MySQLdatabase.SetMaxIdleConns(0)
+	MySQLdatabase.SetMaxOpenConns(500)
```

Comment:
use the database package instead

Reply:
Same response as https://github.com/inverse-inc/packetfence/pull/3024#discussion_r175586997